client/utils/axios.js

```diff
@@ -1,5 +1,21 @@
 import Axios from 'axios';
 
-export default Axios.create({
+const axios = Axios.create({
   baseURL: '/api/v1/'
 });
+
+axios.interceptors.request.use(function (config) {
+  try {
+    const token = JSON.parse(window.localStorage.getItem('auth')).token;
+    if (token) {
+      config.headers = {
+        access_token: token
```

Comment:
Codacy found an issue: [Identifier 'access_token' is not in camel case.](https://app.codacy.com/manual/sam-spain/full-stack-js/pullRequest?prid=6207934)